Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -755,6 +755,18 @@ struct WebCore::ApplePayCouponCodeUpdate : WebCore::ApplePayDetailsUpdateBase {
 };
 #endif
 
+header: <WebCore/ScreenOrientationLockType.h>
+enum class WebCore::ScreenOrientationLockType : uint8_t {
```

Comment:
If you do this, then you should be able to drop the `EnumTraits<WebCore::ScreenOrientationLockType>` in ScreenOrientationLockType.h. They both achieve the same thing.

Reply:
Done. However, I was a bit unsure if these generated things can used without needing ScreenOrientationLockType.h at all?